bedrock/mozorg/redirects.py

```diff
@@ -561,4 +561,6 @@ def decider(request, **kwargs):
     redirect(r"/VendorDPA/?$", "https://assets.mozilla.net/pdf/VendorDPA.pdf", re_flags="i"),
     # Issue 14221
     redirect(r"^firefox/products/?$", "/products/"),
+    # Issue 14255
+    redirect(r"^rise25/?$", "/rise25/nominate/"),
```

Comment:
**nitpick:** would be nice to add a test for this redirect before merging

Reply:
doh, added